build-system/tasks/release/index.js

```diff
@@ -34,7 +34,8 @@ const BASE_FLAVOR_CONFIG = {
   name: 'base',
   rtvPrefixes: ['00', '01', '02', '03', '04', '05'],
   // TODO(#28168, erwinmombay): relace with single `--module --nomodule` command.
-  command: 'gulp dist --noconfig --esm && gulp dist --noconfig',
+  command:
+    'gulp dist --noconfig --esm && gulp dist --sxg && gulp dist --noconfig',
```

Comment:
Is it your intent to add `AMP_CONFIG` to sxg binaries? If not, use `--noconfig`.

Reply:
I'll use `--noconfig`.